tabulator.js

```diff
@@ -1260,15 +1260,32 @@
 
 		var top = typeof top == "undefined" ? self.options.addRowPos : (top === true || top === "top" ? "top" : "bottom");
 
+		// initial place to append the row is the table
+		var appendPlace = self.table;
```

Comment:
please change ***appendPlace*** variable name to ***newRowPosition*** for clarity